### ISSUE TITLE
Address heading level skipping through admin

### ISSUE THREAD

**Issue by kimisgold** [omeka/omeka-s#2382]
This issue is a place to track instances throughout the admin interfaces where heading levels are skipped. Once these areas are identified, we can discuss remediating these patterns. We should watch out especially for instances where parts of the interface are loaded in asynchronously and may not be consistent with the page context, sidebars being the instance that first comes to mind.

**Comment by aviannamiller** [comment]
- In the header nav, section headings should likely be `<h2>`. I would think the sites anchor should also gain that tag, while any nested site anchors would then become `<h3>`.
- Sidebar headings are `<h3>`, should be `<h2>` to avoid a gap between them and `<h1>` page titles.
